index.js

```diff
@@ -197,7 +199,37 @@ function init (ssb, config) {
       const data = keystore.group.get(id)
       if (!data) return cb(new Error(`unknown groupId ${id})`))
 
-      cb(null, data)
+      // find if this group has any parent links
+      scuttle.link.findGroupBySubgroupId(id, (err, links) => {
+        if (err) return cb(err)
+
+        if (!links || !links.length) return cb(null, data) // not a subgroup
+
+        const groupData = {
+          ...data,
+          parentGroupId: links[0].parentGroupId // NOTE: here we assume that there can only be one parent group
+        }
+
+        cb(null, groupData)
+      })
+    })
+  }
+
+  const tribeList = (cb) => {
+    onKeystoreReady(() => {
+      pull(
+        pull.values(keystore.group.list()),
+        pull.asyncMap((groupId, cb) => {
```

Comment:
if `groupId` is included in the results of tribeGet then this can just be `pull.asyncMap(tribeGet)`

Reply:
Its not included, thats why i did this instead

Reply:
Well it is included, but its a buffer